src/app/admin/components/learning-object-list-item/learning-object-list-item.component.ts

```diff
@@ -72,14 +72,14 @@ export class LearningObjectListItemComponent implements OnChanges {
   private headers = new HttpHeaders();
   constructor(
     private auth: AuthService,
-    private unreleaseService: UnreleaseService,
     private statuses: StatusDescriptions,
     private cd: ChangeDetectorRef,
     private http: HttpClient,
     private toaster: ToastrOvenService,
     private hierarchyService: HierarchyService,
     private loService: LearningObjectService,
-    private refactoredLearningObjectService: RefactoredLearningObjectService
+    private refactoredLearningObjectService: RefactoredLearningObjectService,
```

Comment:
nit/debt: remove `refactoredLearningObjectService` and replace all instances of it with `loService`